rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.shards/10_basic.yml

```diff
@@ -267,3 +267,49 @@
 
   - match:
       $body: /foo \s+ 1\n/
+---
+"Test cat shards with change in idle state":
+  - requires:
+      cluster_features: [ "gte_v8.16.0" ]
+      reason: search idle state was added in 8.16.0
+
+  - do:
+      indices.create:
+        index: foo
+        body:
+          settings:
+            number_of_shards: "1"
+            number_of_replicas: "0"
+            search.idle.after: "1s"
+
+  - do:
+      index:
+        index: foo
+        body: { test: foo }
+
+  - do:
+      cat.shards:
+        index: foo
+        v: false
+        h: refresh.is_search_idle
+
+  - match:
+      $body: |
+        /^(false \n?)$/
+
+# very hacky way of sleeping
+  - do:
+      catch: request_timeout
+      cluster.health:
+        wait_for_nodes: 10
+        timeout: "2s"
```

Comment:
Yeah I don't think we need this hack (and I worry that it'll be quite a flaky test in practice). In a YAML test we really just need to verify that the column exists at all.

Reply:
👍
The first test in this file checks for that I believe